hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -245,42 +247,50 @@ protected DBStoreBuilder addOMTablesAndCodecs(DBStoreBuilder builder) {
    */
   protected void initializeOmTables() throws IOException {
     userTable =
-        this.store.getTable(USER_TABLE, String.class, VolumeList.class);
+        this.store.getTable(USER_TABLE, String.class, VolumeList.class,
```

Comment:
This is one argument for why we should not have partial/full distinction. 🙂 It makes initialization of the tables a little clunky.

Reply:
As for some tables where SCM does not need to have cache, so that is the reason for introducing CacheType.
And also for some tables like deletedtable, where this is used by background thread inOM, these does not need cache.

And as for volume/bucket table we plan to have entire table info, and cleanup for them is no-op, even after flush to db, where as for partial cache the cache entries will be cleaned up. This is the only difference between full table cache vs partial table cache.

Reply:
Done. Now caller's no need to specify CacheType.